examples/demo-vendoring/stacks/deploy/staging.yaml

```
@@ -0,0 +1,12 @@
+vars:
+  stage: staging
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)